### ISSUE TITLE
remediation incomplete

### ISSUE THREAD

**Issue by kkrsharma** [ZeroM3m0ry/Btc-Grabber#2]
download failed 

Detected: Trojan:Win32/Conteban.A!ml

what to do?

**Comment by ZeroM3m0ry** [comment]
this is false positive alert i've noticed that some antiviruses flagged the shellcode as malicious , the solution is to turn off or exclude the project folders/file from antivirus .
